index.html

```diff
@@ -216,6 +244,7 @@ <h2><code>PointerEvent</code> interface</h2>
     boolean     isPrimary = false;
     sequence&lt;PointerEvent> coalescedEvents = [];
     sequence&lt;PointerEvent> predictedEvents = [];
+    DeviceProperties deviceProperties;
```

Comment:
Hmm, this doesn't quite work. What if deviceProperties is not passed, what is the deviceProperties value in the PointerEvent. I think we need to let this one to be null and also the attribute on the PointerEvent

Reply:
This has nothing to do with uniqueId. The question is that if one calls
let e = new PointerEvent("foo");
What is e.deviceProperties? Given the webidl in the PR, it can't be null, but nothing explains what it is.
That wpt seems to test behavior which isn't defined.

Reply:
Thanks for your prompt response! I think I was misunderstanding WebIDL and the function of that line- I assumed it would be instantiated with the default DeviceProperties constructor rather than undefined. I'm made the change - I'll update the test as well.

Reply:
Please let me know if you have any more questions or feedback.